qemu/tests/cfg/usb.cfg

```diff
@@ -338,6 +338,51 @@
                 - usb_multi_times:
                     no usb_negative_test
                     usb_repeat_times = 20
+        - usb_redir:
+            type = usb_redir
+            only Linux
+            no Host_RHEL.m8.u3
+            usb_devices += " usbredir1"
+            usb_type_usbredir1 = usb-redir
+            chardev_id_usbredir1 = usbredir_chardev1
+            chardev_backend_usbredir1 = spicevmc
+            chardev_name_usbredir1 = usbredir
+            usbdev_option_chardev_usbredir1 = usbredir_chardev1
```

Comment:
Depends on the comment in vt, add chardev and usb-redir in pair can get it automatically, no need to specify the `chardev` option for the usb device.

Reply:
Updated, thanks!